### PR TITLE
NEWS: update news for 2.0.0 rc2

### PR DESCRIPTION
**Pull request by hppritcha**
@jsquyres 

Signed-off-by: Howard Pritchard howardp@lanl.gov

### CODE REVIEW

**Comment by rhc54** [comment]
I still owe you some text about the async modex option. Also, I think we need to highlight that the dynamic add procs cutoff has been defaulted to zero, and give a very brief statement as to what that means to the user (i.e., less memory but not much else)

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ompi-release-pr/1328/ for details.

File: NEWS [hunk 1]
```diff
@@ -177,6 +177,15 @@ v2.0.0 final release; see:
   Daryin for reporting this issue.
 - Fix various a compilation problems on Cygwin.  Thanks to Marco Atzeri
   for supplying these fixes.
+- Fix for opal pkg-configure file.  Thanks to Mark Santcroos for reporting.
+- Fix a problem with MPI_Win_allocate.  Thanks to Peter Wind for reporting.
```
Comment (jsquyres) [inline NEWS]: Probably should be in all caps to be language neutral (MPI_WIN_ALLOCATE), like you did with MPI_COMBINER_RESIZED, below.

Also, in prior NEWS groups, we tended to put the items in (more or less) reverse-chronological order.  I don't know if that really matters, but that's the way it was (generally) done before.

**Comment by hppritcha** [comment]
i'll move the new entries up to the top of the "fixed with user help" items.

**Comment by hppritcha** [comment]
moved the new entries up to the top of user reporting/fixing section.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ompi-release-pr/1351/ for details.

File: NEWS [hunk 2]
```diff
@@ -109,6 +109,14 @@ v2.0.0 final release; see:
 - Remove support for Myrinet/MX.
 - Remove legacy collective module:ML.
 - Remove support for Alpha processors.
+- Fix for opal pkg-configure file.  Thanks to Mark Santcroos for reporting.
```
Comment (jsquyres) [inline NEWS]: Sorry to be nit-picky: it's pkg-config, right?  (not pkg-configure)

File: NEWS [hunk 3]
```diff
@@ -109,6 +109,14 @@ v2.0.0 final release; see:
...
+- Fix a problem with MPI_Win_allocate.  Thanks to Peter Wind for reporting.
```
Comment (jsquyres) [inline NEWS]: Sorry again to be nit picky... should be MPI_WIN_ALLOCATE (all caps).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ompi-release-pr/1357/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ompi-release-pr/1358/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ompi-release-pr/1391/ for details.

**Comment by hppritcha** [comment]
@jsquyres please double check

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ompi-release-pr/1555/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ompi-release-pr/1556/ for details.

**Comment by hppritcha** [comment]
this got borked by a newer commit. closing

**Pull request closed without merging**